packages/fakta-medisinsk-vilkår/src/ui/MainComponent.stories.tsx

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable no-console */
+import type { Meta, StoryObj } from '@storybook/react';
+import { expect, userEvent, waitFor, within } from '@storybook/test';
+import React, { useEffect } from 'react';
+import { mockUrlPrepend } from '../../mock/constants';
+import { handlers } from '../../mock/handlers';
+import BehandlingType from '../constants/BehandlingType';
+import FagsakYtelseType from '../constants/FagsakYtelseType';
+import MainComponent from './MainComponent';
+
+const meta: Meta<typeof MainComponent> = {
+  title: 'fakta/fakta-medisinsk-vilkår',
+  component: MainComponent,
+  args: {
+    data: {
+      endpoints: {
+        vurderingsoversiktKontinuerligTilsynOgPleie: `${mockUrlPrepend}/mock/kontinuerlig-tilsyn-og-pleie/vurderingsoversikt`,
+        vurderingsoversiktBehovForToOmsorgspersoner: `${mockUrlPrepend}/mock/to-omsorgspersoner/vurderingsoversikt`,
+        dokumentoversikt: `${mockUrlPrepend}/mock/dokumentoversikt`,
+        dataTilVurdering: `${mockUrlPrepend}/mock/data-til-vurdering`,
+        innleggelsesperioder: `${mockUrlPrepend}/mock/innleggelsesperioder`,
+        diagnosekoder: `${mockUrlPrepend}/mock/diagnosekoder`,
+        status: `${mockUrlPrepend}/mock/status`,
+        nyeDokumenter: `${mockUrlPrepend}/mock/nye-dokumenter`,
+        vurderingsoversiktLivetsSluttfase: `${mockUrlPrepend}/mock/livets-sluttfase/vurderingsoversikt`,
+      },
+      behandlingUuid: '123',
+      readOnly: false,
+      onFinished: () => console.log('Aksjonspunkt løst'),
+      visFortsettknapp: true,
+      fagsakYtelseType: FagsakYtelseType.PLEIEPENGER,
+      behandlingType: BehandlingType.FORSTEGANGSSOKNAD,
+      httpErrorHandler: undefined,
+    },
+  },
+  parameters: {
+    test: {
+      dangerouslyIgnoreUnhandledErrors: true,
+    },
+  },
+  decorators: [
+    Story => {
+      useEffect(() => () => window.location.reload(), []);
+      return <Story />;
+    },
+  ],
```

Comment:
Eg testa å ta vekk denne decorator, og det ser ut til å fungere fint uten. Er det ein grunn til at den er her som eg ikkje ser?

Reply:
Det brukes for å resette mockdataene i msw. Dersom jeg tar bort denne, bytter story og bytter tilbake igjen kjøres ikke testen på nytt fordi msw returnere data for fullført gjennomgang

Reply:
Forstår. Ser ut til at det uansett fungerer på fakta-etablert-tilsyn, men det feiler viss ein tek den vekk på fakta-medisinsk-vilkår.

Kanskje det ville vere lurt å lage ein "reusable" decorator funksjon som kan brukast i fleire stories då. Kan og vere meir forklarande til kva grunnen til å bruke den er då. Men det kan vi ta i ein seinare PR. 👌

Reply:
Godt forslag. Jeg lager en PR nå med den endringen